Jenkinsfile

```diff
@@ -107,8 +107,7 @@ pipeline {
                         }
                     }
                     steps {
-                        sh "rm iof.conf"
-                        sconsBuild clean: "_build.external"
+                        sconsBuild(clean: '_build.external iof.conf')
```

Comment:
This actually already is (was) [being done](https://github.com/daos-stack/pipeline-lib/blob/b1ecd6150d74519b5e6d5d3a1caaa9553915af12/vars/sconsBuild.groovy#L131) in [sconsBuild](https://github.com/daos-stack/pipeline-lib/blob/b1ecd6150d74519b5e6d5d3a1caaa9553915af12/vars/sconsBuild.groovy) and is fallout from the removing of the arch in IOF.

This will be fixed when daos-stack/pipeline-lib#29 lands.

Reply:
daos-stack/pipeline-lib#29 has landed.

Reply:
Yes, I saw.  This commit has already been landed however.